src/main/java/com/code_intelligence/jazzer/utils/ZipUtils.java

```diff
@@ -96,4 +100,30 @@ public FileVisitResult visitFile(Path file, BasicFileAttributes attrs) throws IO
 
     return filesAdded;
   }
+
+  public static void extractFile(String srcZip, String targetFile, String outputFilePath)
+      throws IOException {
+    OutputStream out = new FileOutputStream(outputFilePath);
+
+    FileInputStream fis = new FileInputStream(srcZip);
+    BufferedInputStream bis = new BufferedInputStream(fis);
+    ZipInputStream zin = new ZipInputStream(bis);
+
+    ZipEntry ze = zin.getNextEntry();
+    while (ze != null) {
```

Comment:
nit: is it worth it to also check that the file size != 0?

Reply:
I think for a generic extract function we should not do that check